paddle/fluid/operators/mkldnn/doc/data_transformation/data_transform.md

```diff
@@ -0,0 +1,35 @@
+# Design Doc: MKL-DNN Data Transformation
+
+When fluid is using MKL-DNN engine to execute program, not all operators are having mkl-dnn kernels and some of operators are executed by CPU. MKL-DNN kernels of operators expect input Tensors to be provided in MKL-DNN layout , while Paddle(CPU) kernels are expecting input Tensors to be of Paddle layout.
+
+We can distinguish following scenarios(presented below on the picture):
+* Paddle(CPU) kernel is followed by MKL-DNN kernel
+* MKL-DNN kernel is followed by Paddle(CPU) kernel
+* MKL-DNN kernel is followed by fetch operator
+
+
+![](images/data_transform.svg)
```

Comment:
https://github.com/PaddlePaddle/Paddle/blob/63bca64fa29a1b2e719ce47097c9fad57f050d6d/paddle/fluid/operators/mkldnn/doc/data_transformation/data_transform.md I could not see the picture here.

Reply:
@luotao1 I have used SVG as it rendering it should be better quality than PNG. Let me know if there is a need to switch to PNG images

Reply:
No, this PR LGTM!